bundles/org.openhab.binding.bluetooth.bluegiga/src/main/java/org/openhab/binding/bluetooth/bluegiga/internal/BlueGigaSerialHandler.java

```diff
@@ -39,22 +42,32 @@
  * queuing of of data, and conversion of packets from the serial stream into command and response classes.
  *
  * @author Chris Jackson - Initial contribution and API
+ * @author Pauli Anttila - Message correlation
  *
  */
 public class BlueGigaSerialHandler {
 
     private static final int BLE_MAX_LENGTH = 64;
-    private static final int TRANSACTION_TIMEOUT_PERIOD = 50;
+    private static final int TRANSACTION_TIMEOUT_PERIOD_MS = 100;
```

Comment:
Does the bluegiga spec give any guide as to what number this should be?

Reply:
No, at least not the specs which I have found.